feat: add all variants of login page

DawidHaratyk opened the pull request:
closes #53

k1eu reviewed (changes requested):
https://github.com/Selleo/design-system/issues/53
- There's no card component created
- Please extract each icon from the views into own files as we already do with the other icons

k1eu commented:
<img width="746" alt="image" src="https://user-images.githubusercontent.com/32620457/222299776-e562e2eb-0c8b-4d46-884d-db014d13a661.png">
Google version has twitter icon

k1eu commented on packages/selleo-design-core/src/LoginView.tsx:
```diff
@@ -0,0 +1,254 @@
+            <Button
+              variant="primary"
+              size="normal"
+              class="w-[100%] text-center bg-indigo-600 mt-3 py-1 text-white"
```
please don't override button colors

DawidHaratyk replied:
You're right, I'll change that

PoslinskiNet commented:
@DawidHaratyk 

`Sign in to your account` -> `ApplicationName`
`Twitter icon` + "Text" -> `Google Icon + "Sign in"`

Also - why we are not using our buttons from the design system (orange one)?

DawidHaratyk commented:
> 

I've implemented these changes now, about buttons from a design system, I've implemented these as well

DawidHaratyk commented:
> @DawidHaratyk
> 
> `Sign in to your account` -> `ApplicationName` i.e. Dashboard `Twitter icon` + "Text" -> `Google Icon + "Sign in"` `Text about trial` -> `Short description of the application` i.e. Coworker Portal
> 
> Also - why we are not using our buttons from the design system (orange one)?

I've implemented these changes now, about buttons from a design system, I've implemented these as well

k1eu commented on packages/selleo-design-core/src/Button.tsx:
```diff
@@ -1,54 +1,63 @@
       <div class={contentClasses}>
         {IconStart && <IconStart width={iconSize} height={iconSize} />}
-        Text
+        {children}
```
it does break on Button page due to the missing content, can you try to find a way to fix it ? :D

DawidHaratyk replied:
Yes, if there is no text content provided, default text appear
{children || "Text"}
Have you that kind of solution on mind? :D

k1eu replied:
if it works like that sure :D but we have seen issues with the `children` props so I'm asking :D

DawidHaratyk replied:
Now in a button component "children" are rendering as expected :)

k1eu commented on packages/selleo-design-core/src/Button.tsx:
```diff
@@ -1,54 +1,63 @@
+        {children}
         {IconEnd && <IconEnd width={iconSize} height={iconSize} />}
-        {iconTest ? iconTest : ""}
+        {iconTest ? iconTest : ''}
```
iconTest || null

DawidHaratyk replied:
Right, it's better :)

k1eu commented on packages/selleo-design-core/src/LoginView.tsx:
```diff
@@ -0,0 +1,156 @@
+
+export function LoginWithPasswordView() {
+  return (
+    <Card>
+      <div className="flex min-h-full items-center justify-center py-12 px-4 sm:px-6 lg:px-8">
```
Can that be merged, so Card Just takes those styles as classaName?

DawidHaratyk replied:
Yes, I'll merge it, I've just added possibility to add additional styles to the Card component and it takes className as a prop to add own styles :D
